Makefile

```diff
@@ -123,6 +99,15 @@ else
 GOIMPORTS=$(shell which goimports)
 endif
 
+# find or download ko, download ko if necessary
+ko:
+ifeq (, $(shell which ko))
+	GO111MODULE=off go get github.com/google/ko/cmd/ko
```

Comment:
Can we force a preferred version?

Reply:
we could, but it's not clear that we should. The only tag in the repo is from march and that's before go module support was added. The docs for `ko` suggest installing the latest.